### PR TITLE
Changed default behavior of DataFrameBuilder to add a gene column

### PR DESCRIPTION
**Pull request by iskandr**
* Also added missing allele-counts script

---
This change is [Reviewable](https://reviewable.io/reviews/hammerlab/isovar/37)

### CODE REVIEW

File: isovar/dataframe_builder.py [hunk 1]
```diff
@@ -133,6 +146,9 @@ def add(self, variant, element):
                 name = self.rename_dict[name]
             self.columns_dict[name].append(value)
 
+        for column_name, fn in self.extra_column_fns.items():
+            self.columns_dict[column_name] = fn(variant, element)
```
Comment (tavinathanson) [inline isovar/dataframe_builder.py]: Would it simplify the code to make `"gene"` an `extra_column_fn`? A little weird to have `chr/pos/ref/alt` + `gene` (one of these things is not like the other)?

**Comment by tavinathanson** [comment]
LGTM % comment

File: isovar/dataframe_builder.py [hunk 2]
```diff
@@ -55,8 +56,14 @@ def __init__(
             Dictionary mapping element_class field names to desired column names
             in the produced DataFrame.
 
+        extra_column_fns : dict
+            Dictionary mapping column names to functions which take a variant
+            and element and return a single value for each row.
```
Comment (tavinathanson) [inline isovar/dataframe_builder.py]: What does `element` mean here? Maybe elaborate in the comment?

**Comment by ihodes** [comment]
And, to confirm, the extra columns arg is also additional behavior? LGTM

File: isovar/dataframe_builder.py (hunk 2 above)
Comment (iskandr) [inline isovar/dataframe_builder.py]: In every use of a DataFrameBuilder there's some `element_class` (which is an argument to the constructor) such as `AlleleRead`. Maybe I'll add that example to the comment.

File: isovar/dataframe_builder.py (hunk 1 above)
Comment (ihodes) [inline isovar/dataframe_builder.py]: I agree with this, but not sure what the `fn` would be in this case? Could also just add another flag to the function for a `gene` columns, but that starts to get messy.

Comment (iskandr) [inline isovar/dataframe_builder.py]: That was the original motivation for `extra_column_fn` but then I ended up adding it everywhere. Any idea for reducing repetition?

**Pull request merged**